docs/METRICS.md

```diff
@@ -64,6 +67,68 @@ This document outlines all Prometheus metrics exposed by NVSentinel components.
 
 ---
 
+## Node Drainer Module
+
+### Event Processing Metrics
+
+| Metric Name | Type | Labels | Description |
+|------------|------|--------|-------------|
+| `node_drainer_events_received_total` | Counter | - | Total number of events received from the watcher |
+| `node_drainer_events_replayed_total` | Counter | - | Total number of in-progress events replayed at startup |
+| `node_drainer_events_successfully_processed_total` | Counter | - | Total number of events successfully processed |
+| `node_drainer_processing_errors_total` | Counter | `error_type` | Total number of errors encountered during event processing |
+| `node_drainer_event_handling_duration_seconds` | Histogram | - | Histogram of event handling durations |
+| `node_drainer_queue_depth` | Gauge | - | Total number of pending events in the queue |
+
+### Health Event Metrics
```

Comment:
what value do these metrics provide? If there is no value let's drop them. If we really need them, can't we add them as labels to `node_drainer_events_received_total` and `node_drainer_events_replayed_total`?